.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: android
+
+android:
+  components:
+    - build-tools-28.0.3
+    - android-28
+
+jobs:
+  include:
+    - stage: compile
+      language: android
+      before_script: chmod +x gradlew
+      script: ./gradlew assembleDebug assembleRelease
```

Comment:
I'd add 'clean' before 'assembleDebug' just for good measure

Reply:
good suggestion, clean actually erases the build dir ( previous builds )